source/index.html.md.erb

```diff
@@ -3030,11 +3067,7 @@ String temporaryUrl = ss.imageUrlResources().getImageUrls(imageUrls).getImageUrl
 
 Gets a list of URLs that can be used to retrieve the specified cell images.
 
-<aside class="notice">The HTTP verb for this operation is <code>POST</code> (to provide support for specifying request body).</aside>
-
-<aside class="notice">The number and sequence of <a href="#imageurl-object">ImageUrl Objects</a> in the Response corresponds exactly to the number and sequence of objects specified by the Request.</aside>
-
-<aside class="notice">If an error is encountered for any of the requested images, the corresponding <b>ImageUrl</b> Object in the Response contains an <b>error</b> attribute that describes the error. (See Example Response.)</aside>
+<aside class="notice">If an error is encountered for any of the requested images, the corresponding <b>ImageUrl</b> Object in the Response contains an <b>error</b> attribute that describes the error. (See Example Response.) In addition, some errors only appear when you try to reference the URL.</aside>
```

Comment:
I always try to make things as concrete as possible. How about "Additional error checking is performed when the returned URL is referenced." or "Certain errors are not detected until the returned URL is referenced."

Reply:
Done.